.github/actions/post-metric/action.yml

```diff
@@ -0,0 +1,51 @@
+name: Post Datadog Metric
```

Comment:
The filename `post-metric` sounds a little bit too much like "something to be run after the 'metric' step", to my ears. Could we name it "publish-metric" or "submit-metric" instead? And then change this name to "Publish Datadog Metric"?